_posts/fr/2022-09-29-mission-accessible.md

```diff
@@ -0,0 +1,206 @@
+ 
+Une dynamique parfaitement compréhensible et flexible est nécessaire pour se conformer à ce niveau. Le point unique en (O) indique que l'application peut être entièrement naviguée au clavier sans exception, mais il existe de nombreuses spécifications autour de la section Compréhensible (_Understandable_) pour gérer, par exemple, la soumission de données dans un formulaire : il doit être modifiable , appliquer une vérification lors de l'achèvement de la saisie afin de fournir la possibilité de corriger les erreurs et également d'afficher une confirmation avant la soumission.
+ 
+### Vers quel niveau viser ?
+ 
+Tout d'abord, le fait que tous les critères doivent être réunis pour atteindre un niveau **ne doit décourager aucun produit web**. Les efforts doivent être mesurés en fonction du public cible et du type de site Web. Parfois, il y aura des lois qui peuvent pénaliser l'institution ou l'entreprise si elles ne sont pas respectées. Parfois, les frontières sont moins définies et la rigueur est plus lâche.
+ 
+D'autre part, de nombreux critères d'accessibilité ne font que suivre les bonnes pratiques. Cela rend notre code plus propre et plus performant. Cela rend notre conception plus claire et améliore notre UX. Ensuite, et probablement le plus important de tout, **un Web plus accessible rend l'Internet plus démocratique et plus juste**. Alors pourquoi minimiser l'effort ?
+ 
+## Comment évaluer son produit ?
+ 
+Une fois les principes appliqués se pose la question de l’évaluation. Est-ce que votre produit est effectivement davantage accessible pour les contextes d’usages que vous envisagiez ?
+ 
+<div style="text-align: center; margin: 2rem 0;">
+   <img src="{{ site.baseurl }}/assets/2022-09-29-mission-accessible/guidelines.png" width="300px" alt="Illustration of a guidelines book" style="display: block; margin: auto;"/>
+</div>
+ 
+### Outils d’audit
+ 
+Il existe de nombreux outils qui peuvent être utiles pour évaluer le niveau d’accessibilité de votre produit. Parmi eux, nous pouvons citer les outils suivant qui permettent d’évaluer:
+ 
+-   L’accessibilité globale de votre produit et d’identifier les éventuels problèmes d’accessibilité avec [CodeSniffer](https://github.com/squizlabs/HTML_CodeSniffer), [LightHouse](https://developer.chrome.com/docs/lighthouse/overview/) ou [Accessibility Insights](https://accessibilityinsights.io/);
+-   L’utilisation de vos tags HTML en retirant le CSS de votre page avec [Naked Styles](https://gist.github.com/estudiobold/4181e56129ed5cbd0b6cf40c73787a56);
+-   ou encore l’accessibilité de vos couleurs avec [WhoCanUse](https://whocanuse.com/) et [Contrast Grid](http://contrast-grid.eightshapes.com).
+ 
+Cette liste ne représente qu’une fine partie de l’ensemble des outils d’audit d’accessibilité existants, vous pouvez retrouver une liste plus détaillée d’outils recommandés par le W3 via [ce lien](https://www.w3.org/WAI/ER/tools/).
```

Comment:
dans le partie "Les principes POUR selon WCAG" tu parles du W3C. par soucis d'homogénéité et de constance, utilise W3C


```suggestion
Cette liste ne représente qu’une fine partie de l’ensemble des outils d’audit d’accessibilité existants, vous pouvez retrouver une liste plus détaillée d’outils recommandés par le W3C via [ce lien](https://www.w3.org/WAI/ER/tools/).
```

Reply:
done